io.github.masterbillwoo.RiseOfThePenguinsGB.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.masterbillwoo.RiseOfThePenguinsGB
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+command: rise-of-the-penguins-gb-wrapper
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+
+modules:
+  - name: rise-of-the-penguins-gb
+    buildsystem: simple
+    build-commands:
+      # Install executable (architecture-specific name)
+      - |
+        if [ "$(uname -m)" = "x86_64" ]; then
+          install -Dm755 "rise of the penguins gb.x86_64" /app/bin/rise-of-the-penguins-gb
+        else
+          install -Dm755 "rise of the penguins gb.aarch64" /app/bin/rise-of-the-penguins-gb
+        fi
+      - install -Dm644 "rise of the penguins gb.pck" /app/bin/rise-of-the-penguins-gb.pck
+      # Install GodotBoy addon (library and gdextension file)
+      - mkdir -p /app/bin/addons/godot_boy
+      - install -Dm644 addons/godot_boy/godot_boy.gdextension /app/bin/addons/godot_boy/godot_boy.gdextension
+      - |
+        if [ "$(uname -m)" = "x86_64" ]; then
+          install -Dm755 addons/godot_boy/libgodot_boy.linux.x86_64.so /app/bin/addons/godot_boy/libgodot_boy.linux.x86_64.so
+        else
+          install -Dm755 addons/godot_boy/libgodot_boy.linux.aarch64.so /app/bin/addons/godot_boy/libgodot_boy.linux.aarch64.so
+        fi
+      # Create wrapper script that runs from /app/bin
+      - |
+        cat > /app/bin/rise-of-the-penguins-gb-wrapper << 'WRAPPER'
```

Comment:
Never use shell commands to generate a script. Use `type: script` or directly a file.